include/spdlog/logger.h

```diff
@@ -147,6 +147,19 @@ class SPDLOG_API logger
         log(loc, lvl, string_view_t{msg});
     }
 
+    void log(log_clock::time_point &log_time, source_loc loc, level::level_enum lvl, string_view_t msg)
```

Comment:
I think log_time should passed by value here

Reply:
I agree. That's a mistake. I know I had it once as a ref, but I thought I had changed it everywhere.
I missed this one. Sorry about this.
I pushed a change to ron003:single_log_method_with_time_point
Is that the proper way to rectify?
Thanks